CMakeLists.txt

```diff
@@ -51,6 +45,8 @@ set(ITK_VERSION
     "${ITK_VERSION_MAJOR}.${ITK_VERSION_MINOR}.${ITK_VERSION_PATCH}")
 project(ITK
   VERSION ${ITK_VERSION}
+  DESCRIPTION "The Insight Toolkit (ITK) is an open-source, cross-platform toolkit for N-dimensional scientific image processing, segmentation, and registration."
+  HOMEPAGE_URL "https://github.com/InsightSoftwareConsortium/ITK"
```

Comment:
Shouldn't `HOMEPAGE_URL` be "https://itk.org"? Just a question, not a request  😃

Reply:
I'm good with either URL. Originally I had https://itk.org but then put in the Github page.   Please let me know what the preference is.

Reply:
Thanks Hans. I'm good with either URL as well, I feel that it isn't really up to me anyway! Just want to be sure that you did consider https://itk.org, and apparently you did  😄